src/tests/p11test/p11test_case_common.c

```diff
@@ -397,8 +435,11 @@ int callback_public_keys(test_certs_t *objects,
 			EC_GROUP_free(ecgroup);
 			return -1;
 		}
-
+#if OPENSSL_VERSION_NUMBER < 0x30000000L
 		ecpoint = EC_POINT_bn2point(ecgroup, bn, NULL, NULL);
+#else
+		ecpoint = EC_POINT_hex2point(ecgroup, BN_bn2hex(bn), NULL, NULL);
```

Comment:
Not really, since EC_POINT_bn2point is removed completely, it is just a bit workaround to use EC_POINT_hex2point with BN_bn2hex, so I just removed branch with EC_POINT_bn2point.